spec/scss_converter_spec.rb

```diff
@@ -230,4 +232,98 @@ def converter(overrides = {})
 
   end
 
+  context "importing from internal libraries" do
+    let(:internal_library) { source_dir("bower_components/jquery") }
+    let(:converter) { site.getConverterImpl(Jekyll::Converters::Scss) }
```

Comment:
This will have to be `site.find_converter_instance` in Jekyll 3 and `site.getConverterImpl` in Jekyll 2. I think we already have a convention for doing this in a version-agnostic way somewhere.